.github/workflows/test.yml

```diff
@@ -3,7 +3,7 @@ name: Test
 on:
   push:
     branches:
-      - master
+      - next
```

Comment:
Was this not updated for a reason?

Reply:
It was an oversight, good to include it now